<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersForm</class>
 <widget class="QWidget" name="ParametersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Blur radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="blurRadiusSlider">
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="colorGroup">
     <property name="title">
      <string>Shadow Color</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="redSlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="redSpinBox">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="greenSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="greenSpinBox">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="blueSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="blueSpinBox">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="offsetGroup">
     <property name="title">
      <string>Shadow Offset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="xOffsetSlider">
        <property name="minimum">
         <number>-30</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="xOffsetSpinBox">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>-30</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="yOffsetSlider">
        <property name="minimum">
         <number>-30</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="yOffsetSpinBox">
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>-30</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hideImageCheckBox">
     <property name="text">
      <string>Hide Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>blurRadiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blurRadiusSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xOffsetSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xOffsetSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xOffsetSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xOffsetSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yOffsetSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yOffsetSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yOffsetSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yOffsetSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
